dist/systemd/obs-sphinx.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description = Open Build Service Sphinx Search Daemon
+BindsTo = obs-api-support.target
+Conflicts = searchd.service
+
+[Service]
+Environment = "RAILS_ENV=production"
+User = wwwrun
+Group = www
+WorkingDirectory = /srv/www/obs/api
+ExecStart = /bin/bash -c "if [ `stat -c '%s' /srv/www/obs/api/config/production.sphinx.conf` -eq 0 ]; then /usr/bin/bundle.ruby2.5 exec rails ts:rebuild; else /usr/bin/bundle.ruby2.5 exec rails ts:start; fi"
+ExecStop = /usr/bin/rails ts:stop
```

Comment:
I think you need bundle exec here as well

Reply:
Fixed.